_markbind/navigation/mainNav.md

```diff
@@ -72,6 +72,8 @@
     * [Cryptography]({{baseUrl}}/contents/security/cryptography.html)
     * [HTTPS]({{baseUrl}}/contents/security/Https.html)
     * [SQL Injection]({{baseUrl}}/contents/security/sqlInjection.html)
+  * Linux 
+    * [Shell]({{baseUrl}}/contents/linux/shell.html)
```

Comment:
Could I propose that this article be classified under Shells/Bash instead? The current classification suggests that shells only exist on Linux.

Reasons:
- Bash is [not the only Linux shell](https://www.slant.co/topics/513/~best-unix-shells). Colloquially, `zsh`/`dash` does see use.
- Bash does not only exist on Linux. It is also the default shell in Mac OS X and can be used on Windows through [MSYS2](https://www.msys2.org/) (used in Git Bash) or [Windows Subsytem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
- Windows has shells too, in the form of [Command Prompt](https://en.wikipedia.org/wiki/Cmd.exe) and [PowerShell](https://en.wikipedia.org/wiki/PowerShell). You can also do shell scripting on Command Prompt and PowerShell (which is as powerful as bash at least).